rwiki/rwiki-tool/tool/src/java/uk/ac/cam/caret/sakai/rwiki/tool/command/SaveCommand.java

```diff
@@ -280,6 +293,16 @@ private void contentChangedDispatch(Dispatcher dispatcher,HttpServletRequest req
 		dispatcher.dispatch(contentChangedPath,request, response);
 	}
 
+	
+	private void groupsErrorDispatch(Dispatcher dispatcher,HttpServletRequest request,
+			HttpServletResponse response) throws ServletException, IOException
+	{
+		ErrorBean errorBean = ErrorBeanHelper.getErrorBean(request);
+		ResourceLoaderBean rlb = ResourceLoaderHelperBean.getResourceLoader(request);
+		errorBean.addError(rlb.getString("save.content_no_groups", "Please select at least one group"));
```

Comment:
I guess you have to use a second parameter by default... If you can't get rid of it please use the same text as the one on the properties.

Reply:
Ok. Thanks